crates/nu-cli/src/print.rs

```diff
@@ -16,7 +16,11 @@ impl Command for Print {
 
     fn signature(&self) -> Signature {
         Signature::build("print")
-            .input_output_types(vec![(Type::Nothing, Type::Nothing)])
+            .input_output_types(vec![
+                (Type::Nothing, Type::Nothing),
+                (Type::Any, Type::Nothing),
+            ])
+            .allow_variants_without_examples(true)
```

Comment:
I think it doesn't work with how the example testing is set up but we should slap a regular `nu!` test on this, cause that was embarassing.

Reply:
i can add an example yup :wink:

Reply:
I don't care about the example as much as a test. (I think the `test_examples` stuff directly evaluates to `Value` so the `print` output would get lost)

Reply:
a test. i meant a test :eyes: 
my bad :laughing: :cry:

Reply:
No probs

Reply:
mm i'm not sure where to put such a test :confused: 
there is nothing in `nu-cli` basically, right?

Reply:
@sholderbach 
is the test from 0fba49f54 ok? :innocent:

Reply:
oooh yeah thanks for the fix on the test, i somehow thought that, as `print`'s output can not be piped, `actual.out` would be empty, but it's not what the *output* is in the tests :eyes: